driver-core/src/main/com/mongodb/internal/async/function/RetryState.java

```diff
@@ -203,7 +203,7 @@ private void doAdvanceOrThrow(final Throwable attemptException,
              */
             if (hasTimeoutMs() && !loopState.isLastIteration()) {
                 previouslyChosenException = createMongoTimeoutException(
-                        "MongoDB operation timed out during a retry attempt",
+                        "Retry attempt timed out.",
                         previouslyChosenException);
             }
```

Comment:
The retry could be triggered not only by transient errors in MongoDB operations but also within the scope of OIDC. Referring to it as a 'MongoDB operation' might lead to confusion among users.